api/info.go

```diff
@@ -15,7 +17,7 @@ import (
 // produce: application/json
 // responses:
 //   200: OK
-func info(w http.ResponseWriter, r *http.Request) error {
+func info(w http.ResponseWriter, r *http.Request, t auth.Token) error {
```

Comment:
Now the CI is failing because this route is authenticated and the test `TestInfo` isn't prepared for.